django_sorting_field/widgets.py

```diff
@@ -15,7 +15,7 @@ class Media:
             "sorting/js/sorting_widget.js",
         )
 
-    def render(self, name, value, attrs=None):
+    def render(self, name, value, attrs=None, renderer=None):
```

Comment:
Doesn't seem to be used..?

Reply:
Nevermind. I guess it's good to keep it here to keep the signature compatible with the parent class implementation.

Reply:
Since Django 2.x calls this with the renderer argument, deviating from the signature will cause an exception. Thanks for the merge.